### PR TITLE
Support single-PLD Q2TIPS PASL and multi-PLD PCASL

### LINKED ISSUE #250: Re-enable multi-PLD PCASL support
## Summary

I blocked any multi-PLD support in #235, because the averaging formula that was being applied was incorrect, and didn't generalize across the different methods (PCASL, PASL+Q2TIPS, PASL+QUIPSS, PASL+QUIPSSII).

However, we would like to get this working at some point soon.

### LINKED ISSUE #251: Support Q2TIPS data
## Summary

I disabled Q2TIPS CBF quantification in #235, because it wasn't being done properly. However, we'd like to get it working in the near future.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #250 and closes #251.

To do:

- [ ] Decide between Juttukonda 2021 unweighted average and Dai 2012 weighted average for multi-PLD PCASL CBF calculation.
    - I don't know how to do the weighted average yet.
- [x] Define T1tissue based on magnet strength.
    - I'm not aware of any papers with formulae, but Manuel Taso pointed me toward [Wright et al. (2008)](https://link.springer.com/article/10.1007/s10334-008-0104-8). This unfortunately limits us to 1.5, 3, and 7T data.
- [x] Double-check the code and formulae.

## Changes proposed in this pull request

- Estimate ATT for multi-PLD PCASL data using the Dai 2012 method. Use this to calculate CBF for those data.
- Use formula from Noguchi 2015 to calculate CBF for single-PLD Q2TIPS data.
- Add license info for all copied, modified, translated, or otherwise external code.
- Update test datasets by removing hidden files.
- Also, fix test_002 dataset:
    - Drop second deltam volume. It's just a duplicate of the first, and was a conversion bug.
    - Add necessary metadata fields, including RepetitionTimePreparation for the M0 scan and LabelingEfficiency, BackgroundSuppression, and BackgroundSuppressionNumberPulses for the asl scan.
- Add modified version of FSL Split interface that enforces a list output.
- Modify `_select_first_in_list` to only return the first item if a list is provided.
- Rename `aslprep.interfaces.cbf_computation` to `aslprep.interfaces.cbf`.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2018, PENNLINC,CRN, NIPREP developers team.
```
Comment (tsalo) [inline LICENSE.md]: @mattcieslak would you mind taking a look at this?

File: aslprep/utils/misc.py [hunk 2]
```diff
@@ -771,3 +787,255 @@ def estimate_labeling_efficiency(metadata):
             labeleff *= BS_PULSE_EFF ** metadata.get("BackgroundSuppressionNumberPulses", 1)
 
     return labeleff
+
+
+def estimate_att_pcasl(deltam_arr, plds, lds, t1blood, t1tissue):
+    """Estimate arterial transit time using the weighted average method.
```
Comment (tsalo) [inline aslprep/utils/misc.py]: @mattcieslak would you mind taking a look at the license info here?

File: LICENSE.md (hunk 1 above)
Comment (mattcieslak) [reply LICENSE.md]: Do we still have an internal version of niworkflows? For code we use externally we can add their license files and copyright notices

File: aslprep/utils/misc.py (hunk 2 above)
Comment (mattcieslak) [reply aslprep/utils/misc.py]: This is great!

File: LICENSE.md (hunk 1 above)
Comment (tsalo) [reply LICENSE.md]: Yes we do, from before the relicensing. Same with sdcflows. I'll add those then.

File: aslprep/utils/misc.py (hunk 2 above)
Comment (tsalo) [reply aslprep/utils/misc.py]: Thanks!

**Pull request merged**